modal/_container_entrypoint.py

```diff
@@ -161,6 +161,12 @@ def serialize(self, obj: Any) -> bytes:
     def deserialize(self, data: bytes) -> Any:
         return deserialize(data, self._client)
 
+    def serialize_data_format(self, obj: Any, data_format: int) -> bytes:
```

Comment:
tiny nit, should this be
```suggestion
    def serialize_data_format(self, obj: Any, data_format: api_pb2.DataFormat.ValueType) -> bytes:
```

Reply:
Honestly don't know how this affects mypy, can I defer it until later?

Reply:
I've had a bunch of mypy issues with `ValueType`